packages/mantine/src/theme/Theme.tsx

```diff
@@ -99,6 +99,19 @@ export const plasmaTheme: MantineThemeOverride = {
             defaultProps: {
                 radius: 8,
             },
+            styles: (theme) => ({
+                description: {
+                    fontSize: 'inherit',
+                    paddingBottom: theme.spacing.xs,
+                },
+                invalid: {
+                    color: theme.colors.red[9],
+                    borderColor: theme.colors.red[6],
+                },
+                error: {
+                    color: theme.colors.red[9],
+                },
```

Comment:
Surely this style should also be applied to other form elements? I'm thinking of the Select for example, I'm certainly forgetting others.

Reply:
I can add the same style logic to the Select component since it's still in the scope. If we find any other components later, we can just change the style the same way

Reply:
Have a look at InputWrapper, its the thing that renders the errors for every form component. If you modify its style it will apply to each of them